docs/block-api.md

```diff
@@ -144,6 +133,13 @@ anchor: true,
 customClassName: false,
 ```
 
+- `generatedClassName` (default `true`): By default, Gutenberg adds a class with the form `.wp-blocks-your-block-name` to the root element of your saved markup. This helps having a consistent mechanism for styling blocks that themes and plugins can rely on. If for whatever reason a class is not desired on the markup, this functionality can be disabled.
```

Comment:
There's an error in the example: it uses `blocks` as plural, should be just `wp-block`.